docs-translations/zh-CN/api/notification.md

```diff
@@ -0,0 +1,109 @@
+# 通知
+
+> 创建系统桌面通知
+
+进程: [Main](../glossary.md#main-process)
+
+## 在渲染器进程中使用
+
+如果你想要从渲染器进程显示通知，则应使用 [HTML5 Notification API](../tutorial/notifications.md)
+
+## 类: Notification
+
+> 创建系统桌面通知
+
+进程: [Main](../glossary.md#main-process)
+
+`Notification` 是一个
+[EventEmitter](http://nodejs.org/api/events.html#events_class_events_eventemitter).
+
+它通过由 `options ` 设置的原生属性创建一个新的 `Notification`.
+
+### 静态方法
+
+`Notification` 类具有以下静态方法:
+
+#### `Notification.isSupported()`
+
+返回 `Boolean` - 无论当前系统是否支持桌面通知
+
+### `new Notification([options])` _实验性_
+
+* `options` 对象
+  * `title` 字符串 - 通知的标题，显示在通知窗口的顶部.
+  * `subtitle` 字符串 - (可选) 通知的副标题，将显示在标题下方.  _macOS_
+  * `body` 字符串 - 通知的正文，将显示在标题或副标题下方.
+  * `silent` 布尔 - (可选) 是否在显示通知时发出系统通知提示音.
+  * `icon` [NativeImage](native-image.md) - (可选) 通知所使用的图标
+  * `hasReply` 布尔 - (可选) 是否在通知中添加内联的回复选项.  _macOS_
+  * `replyPlaceholder` 字符串 - (可选) 在内联输入字段中的提示占位符.  _macOS_
+  * `sound` 字符串 - (可选) 显示通知时要播放的声音文件的名称.  _macOS_
+  * `actions` [NotificationAction[]](structures/notification-action.md) - (可选) 添加到通知中的操作. 请阅读 `NotificationAction` 文档中的可用操作和限制.  _macOS_
```

Comment:
Same! Page not exist

Reply:
https://github.com/electron/electron/pull/10749